### PR TITLE
Changing font-style/size

### LINKED ISSUE #67: Make all typeface on guide articles Lato
We should replace the current font on the guide articles with Lato so it's consistent with the rest of the text on freeCodeCamp. We should also bump the font size up to at least 14px.

### PR DESCRIPTION
**Pull request by vikramnr**
closes #67

### CODE REVIEW

**Comment by vikramnr** [comment]
@Bouncey  I have added the local font files and also via package.  Kindly let me know if anything needs to changed.

File: src/html.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ export default class HTML extends React.Component {
           />
           <title>freeCodeCamp Guide</title>
           <TypographyStyle typography={typography} />
+           <GoogleFont typography={typography} />
```
Comment (Bouncey) [inline src/html.js]: Indenting here. Should be the same nesting level as `Typography` and `title`

File: src/utils/typography.js [hunk 2]
```diff
@@ -3,15 +3,27 @@ import CodePlugin from 'typography-plugin-code';
...
+  baseFontSize: '14px',
   baseLineHeight: 1.45,
   scaleRatio: 2.25,
+   googleFonts: [
+    {
+      name: 'Lato',
+      styles: [
+        '400',
+        '400i',
+        '900',
```
Comment (Bouncey) [inline src/utils/typography.js]: Trailing commas here, we don't use these in this project.

Try running `npm run lint` before pushing to github 👍

**Comment by vikramnr** [comment]
@Bouncey  I'll make those changes 👍

**Comment by Bouncey** [comment]
@vikramnr I can't see where the `.ttf` files are used?

File: src/utils/typography.js [hunk 3]
```diff
@@ -3,15 +3,18 @@ import CodePlugin from 'typography-plugin-code';
...
+  baseFontSize: '14px',
   baseLineHeight: 1.45,
   scaleRatio: 2.25,
+  googleFonts: [name:'Lato'],
+  headerFontFamily: ['Lato', 'serif'],
+  bodyFontFamily: ['Lato', 'serif'],
   plugins: [new CodePlugin()],
   overrideStyles: () => ({
     [MOBILE_MEDIA_QUERY]: {
       // Make baseFontSize on mobile 16px.
```
Comment (Bouncey) [inline src/utils/typography.js]: You need to update the comment also,

File: src/utils/typography.js [hunk 4]
```diff
@@ -3,15 +3,18 @@ import CodePlugin from 'typography-plugin-code';
 import { MOBILE_MEDIA_QUERY } from 'typography-breakpoint-constants';
 
 const options = {
-  baseFontSize: '18px',
+  baseFontSize: '14px',
   baseLineHeight: 1.45,
   scaleRatio: 2.25,
+  googleFonts: [name:'Lato'],
```
Comment (Bouncey) [inline src/utils/typography.js]: This isn't valid javaScript, and with it in place, it breaks the whole site. 

Please make the changes required and test that it works before amending this commit

Comment (vikramnr) [inline src/utils/typography.js]: @Bouncey I'll make the changes

**Comment by vikramnr** [comment]
@Bouncey  I have updated the changes in typography.js and I think Lato is available as default so there is no need for font files and google font tags. Also its not breaking the site as earlier. 
Running in Local

**Comment by Bouncey** [comment]
@vikramnr thanks for making the changes. Do we need the `.ttf` files? Also, you still need to update the comment about font size on mobile

**Comment by vikramnr** [comment]
@Bouncey  We don't need .ttf files. And Will update comments.  This is my first PR so apologies if this is causing any delays

**Comment by vikramnr** [comment]
@Bouncey  From this `// Make baseFontSize on mobile 16px.` `// Make baseFontSize on mobile 14px.` this 

Is that right?

**Comment by Bouncey** [comment]
@vikramnr yes

**Comment by Bouncey** [comment]
@vikramnr Thanks for updating the PR, I will pull this down later tonight and test 👍

**Comment by Bouncey** [comment]
## LGTM :+1: 

Thanks @vikramnr :sparkles: :sparkles:

**Pull request merged**